tests/fixtures/storage/golden_images.py

```
@@ -0,0 +1,138 @@
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
+def golden_images_namespace(
+    admin_client,
+):
+    for ns in Namespace.get(
+        name=py_config["golden_images_namespace"],
+        client=admin_client,
+    ):
+        return ns
```

Ruff (0.16.1)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)